robyn/__main__.py

```diff
@@ -0,0 +1,28 @@
+import os
+import click
+
+
+@click.command()
+def create():
+    project_name = input("Enter the project name: ")
+    project_dir = input("Enter the name of the project directory: ")
+    # Initailize a new Robyn project
+
+    print(f"Creating a new Robyn project '{project_name}' in '{project_dir}'...")
+
+    # Create a new directory for the project
+    project_path = os.path.join(project_dir, project_name)
+    os.makedirs(project_path)
+
+    # Create the main application file
+    app_file_path = os.path.join(project_path, "app.py")
+    with open(app_file_path, "w") as f:
+        f.write(
+            'from robyn import Robyn\n\napp = Robyn()\n\nif __name__ == "__main__":\n app.run()'
```

Comment:
One last change. The syntax should be this 

```
            'from robyn import Robyn\n\napp = Robyn(__file__)\n\nif __name__ == "__main__":\n app.run()'
```

Reply:
@sansyrox  All done